CONTRIBUTING.md

```diff
@@ -275,7 +274,7 @@ docker compose --file ./e2e/docker-compose.yaml down
 
 ### Run Local End-to-End Tests Against Your Dev Server Console
 
-If you also make [changes to the console](#console), you can run the test suite against your locally built backend code and frontend server.
+If you also make [changes to the console](#contribute-console), you can run the test suite against your locally built backend code and frontend server.
```

Comment:
Do we need to prefix the headings with contribute-*, feels weird since the file is about contribute and we could just call it console, login, core, ...

Reply:
I fixed all the anchors